src/components/DataTable/TableToolbarAction.js

```diff
@@ -2,6 +2,7 @@ import cx from 'classnames';
 import PropTypes from 'prop-types';
 import React from 'react';
 import Icon from '../Icon';
+import isRequiredThisOrOneOf from '../../internal/isRequiredThisOrOneOf';
```

Comment:
Want to put this utility in our `prop-types` sub-folder?

Reply:
@joshblack OK done!